source/user-manual/wazuh-dashboard/custom-branding.rst

```diff
@@ -16,13 +16,33 @@ Custom logos in the Wazuh dashboard
 
 To use your own logos in the Wazuh dashboard, click on **Indexer/dashboard management** > **App Settings**. Under the **Custom branding** section, set up the following properties:
 
--  ``customization.logo.app``. This property sets the `App main logo` image. It has a size limit of 1 MB. It replaces the logo image in the Wazuh main menu located top left. Recommended size: 300 pixels width, 70 pixels height. See #1 in the image below.
+-  ``customization.logo.app``. This property sets the `App loading logo` image. It has a size limit of 1 MB. It replaces the logo image in the Wazuh loading animation when a new section initializes. Recommended size: 300 pixels width, 70 pixels height.
 
--  ``customization.logo.healthcheck``. This property sets the `Health check logo` image. It has a size limit of 1 MB. It replaces the logo on top of the check list displayed during the health check routine. Recommended size: 300 pixels width, 70 pixels height. See #2 in the image below.
+.. thumbnail:: /images/kibana-app/features/white-labeling/custom-branding-settings-loading.png
+   :align: center
+   :width: 100%
+
+-  ``customization.logo.healthcheck``. This property sets the `Health check logo` image. It has a size limit of 1 MB. It replaces the logo on top of the check list displayed during the health check routine. Recommended size: 300 pixels width, 70 pixels height.
 
-.. thumbnail:: /images/kibana-app/features/white-labeling/custom-branding-settings.jpg
+.. thumbnail:: /images/kibana-app/features/white-labeling/custom-branding-settings-health-check.png
    :align: center
-   :width: 80%
+   :width: 100%
+
+
+To customize the `Wazuh dashboard home logo` in the top header, you have to edit the file ``/usr/share/wazuh-dashboard/config/opensearch_dashboards.yml`` and add the URL of the default and dark theme logos.
```

Comment:
**nitpick**: The configuration file in the `.rpm`, and `.deb` is located at `/etc/wazuh-dashboard/opensearch_dashboards.yml` (see https://documentation.wazuh.com/current/installation-guide/wazuh-dashboard/step-by-step.html#configuring-the-wazuh-dashboard) while the Docker distribution could have using the `/usr/share/wazuh-dashboard/config/opensearch_dashboards.yml` (see https://github.com/wazuh/wazuh-docker/blob/4.8.0/build-docker-images/wazuh-dashboard/config/entrypoint.sh#L20). We should review it.